Response-Theory/Coupled-Cluster/RHF/optrot.py

```diff
@@ -0,0 +1,281 @@
+#
+# Velocity gauge (VG) representation gives a non-zero optical rotation at zero frequency,
+# which is clearly an unphysical result. Pedersen et al. (ref. 4) proposed the modified
+# velocity gauge (MVG) representation where the VG optical rotation at # zero frequency is subtracted from VG results at a given frequency. 
+
+print("\n\nModified Velocity Gauge Calculations Starting ..\n\n")
+
+Om_0 = str(0)
+for i in range(0,3):
+    L = "L_" + cart[i]
+    P = "P_" + cart[i]
+    Om_0 = str(0)
+    # Initializing perturbation classes at zero frequency    
+    ccpert[L  + Om_0] = HelperCCPert(L,  pert[L],  ccsd, cchbar, cclambda, 0)
+    ccpert[P  + Om_0] = HelperCCPert(P,  pert[P],  ccsd, cchbar, cclambda, 0)
+    # Solving X and Y amplitudes of the perturbation classes at zero frequency
+    print('\nsolving right hand perturbed amplitudes for %s @ omega = %s (a.u.)\n'% (L, Om_0))
+    ccpert[L + Om_0].solve('right', r_conv=1e-10)
+    print('\nsolving left hand perturbed amplitudes for %s @ omega = %s (a.u.)\n' % (L, Om_0))
+    ccpert[L + Om_0].solve('left', r_conv=1e-10)
+    print('\nsolving right hand perturbed amplitudes for %s @ omega = %s (a.u.)\n'% (P, Om_0))
+    ccpert[P + Om_0].solve('right', r_conv=1e-10)
+    print('\nsolving left hand perturbed amplitudes for %s @ omega = %s (a.u.)\n' % (P, Om_0))
+    ccpert[P + Om_0].solve('left', r_conv=1e-10)
+
+
+for A in range(0,3):
+    str_A = "P_" + cart[A]
+    for B in range(0,3):
+        str_B = "L_" + cart[B]
+        str_AB = "<<" + str_A + ";" + str_B + ">>"
+        str_BA = "<<" + str_B + ";" + str_A + ">>"
+        # constructing the linear response functions <<P;L>> and <<L;P>> @ zero frequency)
+        cclinresp[str_AB]= HelperCCLinresp(cclambda, ccpert[str_A + Om_0], ccpert[str_B + Om_0])
+        cclinresp[str_BA]= HelperCCLinresp(cclambda, ccpert[str_B + Om_0], ccpert[str_A + Om_0])
+        tensor[str_AB]= cclinresp[str_AB].linresp()
+        tensor[str_BA]= cclinresp[str_BA].linresp()
+        optrot_vg_0[3*A+B] = 0.5 * (tensor[str_AB] + tensor[str_BA])
+
+#  MVG(omega) = VG(omega) - VG(0)
+optrot_mvg = optrot_vg_om - optrot_vg_0
+
+# Isotropic optical rotation in modified velocity gauge @ given omega 
+rmvg_au = optrot_mvg[0] + optrot_mvg[4] + optrot_mvg[8]
+rmvg_au /= 3 
+
+print('\n CCSD Optical Rotation Tensor (Modified Velocity Gauge) @ %d nm'%omega_nm)
+print("\t\t%s\t             %s\t                  %s\n" % (cart[0], cart[1], cart[2]))
+for a in range(0,3):
+    print(" %s %20.10lf %20.10lf %20.10lf\n" % ( cart[a], optrot_vg_0[3*a+0], optrot_vg_0[3*a+1], optrot_vg_0[3*a+2]))
+
+specific_rotation_mvg = prefactor * rmvg_au/Mass 
+print("Specific rotation @ %d nm (Modified Velocity Gauge): %10.5lf deg/[dm (g/cm^3)]"% (omega_nm, specific_rotation_mvg))
+
+"""#  Comaprison with PSI4 (if you have near to latest version of psi4)
+psi4.set_options({'d_convergence': 1e-10})
```

Comment:
Can yo usen the options as a single block?

```
psi4.set_option({'opt1': '..',
                            'opt2', '..'})

Reply:
Done!